### PR TITLE
[ru] Merge the dev-ru branch into the main branch

### PR DESCRIPTION
**Pull request by kirkonru**
### Describe your changes

Merge the `dev-ru` branch into the `main` branch to update the CNCF glossary with the terms translated into Russian.

### Related issue number or link (ex: `resolves #issue-number`)

- No related issue

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by tym83 — APPROVED** [review]
LGTM

File: content/ru/digital-certificate.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Digital Certificate
```
Comment (shurup) [inline content/ru/digital-certificate.md:2]:
```suggestion
title: Цифровой сертификат
```

File: content/ru/distributed-systems.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: Distributed System
```
Comment (shurup) [inline content/ru/distributed-systems.md:2]:
```suggestion
title: Распределенная система
```

**Review by shurup — CHANGES_REQUESTED** [review]
Happy to see this finally happening! However, I just noticed two small issues we somehow overlooked. Let's fix them before merging.

**Review by shurup — APPROVED** [review]
Now, when my suggestions are applied, LGTM!

**Review by seokho-son — APPROVED** [review]
Looks good to me!! :)

All PRs have been reviewed already. So, on behalf of the localization reviewers, let me merge this PR! 

Also, thanks for the clear commit history! well mangaged!

**Pull request merged**